savoirfairelinux/sous-chef
# Fixed: move Referencing.work_information to Member.work_information

## Fixes #604  by lingxiaoyang

### Changes proposed in this pull request:

1. Migrate the field content. Prompt during migration if a member may have multiple work information in db.
2. Change logic and test.
3. Move the field "work_information" from Referencing to Member in `delivery/fixtures/sample_data.json`.
4. Change the referent form display.
![capture du 2016-12-27 14-04-54](https://cloud.githubusercontent.com/assets/8630726/21506413/0e9d07e6-cc3e-11e6-933f-0489c49ba309.png)


### Status


### How to verify this change

1. Before migration, create a Referencing object in Django admin and set work information.
2. Execute manage.py migrate
3. The work information field content should be moved to the related Member object.
4. Create/Edit an existing client, go to referent form, and the work information field should only show when adding a new member.
5. After saving, the work information should be stored on Member object.

### Deployment notes and migration


### New translatable strings

none

### Additional notes

none
